pkg/controller/perconaservermongodb/secrets.go

```diff
@@ -158,3 +134,42 @@ func (r *ReconcilePerconaServerMongoDB) reconcileUsersSecret(ctx context.Context
 
 	return nil
 }
+
+func fillSecretData(cr *api.PerconaServerMongoDB, data map[string][]byte) (changes bool, err error) {
+	if data == nil {
+		data = make(map[string][]byte)
+	}
+	userMap := map[string]string{
+		envMongoDBBackupUser:         string(roleBackup),
+		envMongoDBClusterAdminUser:   string(roleClusterAdmin),
+		envMongoDBClusterMonitorUser: string(roleClusterMonitor),
+		envMongoDBUserAdminUser:      string(roleUserAdmin),
+	}
```

Comment:
should we use this one 
`MONGODB_DATABASE_ADMIN_USER: databaseAdmin
  MONGODB_DATABASE_ADMIN_PASSWORD: databaseAdmin123456`

Reply:
We have it here https://github.com/percona/percona-server-mongodb-operator/blob/d40a81a9166469829ee2e1a8d6df05c6a67d145b/pkg/controller/perconaservermongodb/secrets.go#L154-L157